mlflow/tracking/fluent.py

```diff
@@ -472,6 +485,47 @@ def delete_tag(key: str) -> None:
     MlflowClient().delete_tag(run_id, key)
 
 
+def set_description(description: str) -> None:
```

Comment:
Let's not add new methods to the fluent API here. 
Having the attribute in start_run() is sufficient to enable setting of the description field. 

Both of these methods can be removed.